### PR TITLE
PHP FILE

### PR DESCRIPTION
**Pull request by Lockf0rc**
HERE IS THE PHP  FILE

### CODE REVIEW

File: Corrections/Corrections_php.php [hunk 1]
```diff
@@ -0,0 +1,254 @@
...
+
+// FIXME: Print
+// Print the following sentence in the console "You are NAME and you are AGE years old !". Don't forget to add a newline at the end
+  
+  echo "You are $name and you are $age years old!";
+  
+// FiXME: Concatenation
+// Create a new string variable called "hello" which value is "Hello ". Add "name" at the end of "hello" (Concatenation) then print it
+
+$hello="hello";
```
Comment (P1kachu) [inline Corrections/Corrections_php.php]: Concatenation and printing are missing.

File: Corrections/Corrections_php.php [hunk 2]
```diff
@@ -0,0 +1,254 @@
...
+
+// FIXME: Array
+// create a new string array called "shoppingList", with three elements of your choice. Create an int variable containing the number of
+// elements in "shoppingList" (using a function of the array/using the array)
+
+$shoppinglist= array("avocados","cheese","fruitloops");
```
Comment (P1kachu) [inline Corrections/Corrections_php.php:54]: Variable containing the length of the array is missing.

File: Corrections/Corrections_php.php [hunk 3]
```diff
@@ -0,0 +1,254 @@
...
+for($i=0;$i<count($shoppinglist);$i++){
+  echo $shoppinglist[$i];
+}
+
+// FIXME: Foreach-loop
+// Do the same with a foreach-loop.
+
+foreach($shoppinglist as $key){
+echo $key
+
```
Comment (P1kachu) [inline Corrections/Corrections_php.php]: "}" Bracket missing (?)

File: Corrections/Corrections_php.php [hunk 4]
```diff
@@ -0,0 +1,254 @@
...
+// Create a string variable called "element" with the value of your choice. Then create a for-loop that checks if "shoppingList" contains 
+// "element". If yes, print "You have to buy (value of element) !", and stop the loop (search how to stop a loop).
+// If not, print "Nope, you don't need (value of "element")".
+
+
+$element="Book Case";
+
+foreach($shoppingList as $item){
+  if($item == $element){
+  echo "You have to buy ( value of $item )!"
```
Comment (P1kachu) [inline Corrections/Corrections_php.php:123]: You have to find a way to stop the loop in that case.

File: Corrections/Corrections_php.php [hunk 5]
```diff
@@ -0,0 +1,254 @@
...
+}
+
+// FIXME: User entry
+// Create a string variable that takes what the user enter in the console as value. Then print "You entered (value of string)"
+
+// FIXME: While loop
+// Create a while loop that takes a number and divides it by 2 until it is less than 3
+
+// FIXME: do-While loop
+// Do the same with a do-while loop
```
Comment (P1kachu) [inline Corrections/Corrections_php.php:235]: I can't seem to find these 3 functions.

File: Corrections/Corrections_php.php (hunk 1 above)
Comment (Razonator) [inline Corrections/Corrections_php.php]: And fix the indentation for more homogeneity

File: Corrections/Corrections_php.php [hunk 6]
```diff
@@ -0,0 +1,351 @@
...
+// FIXME: Functions - Ascii
+// Create a function that returns nothing and which doesn't takes any parameter. It should just be named "TriForce"
+// and print the TriForce symbol (one triangle over two other ones, can be found on internet) with "TRIFORCE"
+// Don't forget to call the function !
+
+/*   http://php.net/manual/en/language.types.string.php */
+function TriForce()
+{
+//  SEE HereDOc  http://www.php.net/manual/en/language.types.string.php#language.types.string.syntax.heredoc
+$string= <<<EOD
```
Comment (Lockf0rc) [inline Corrections/Corrections_php.php:149]: The HEREDOC breaks the syntax highlighter  in GITHUB.

File: Corrections/Corrections_php.php (hunk 2 above)
Comment (Lockf0rc) [inline Corrections/Corrections_php.php:54]: added the array length using  the count function

**Comment by P1kachu** [comment]
Thank you very much for your participation, I will note it in the README !

**Comment by mGm-Lizard** [comment]
This is a cool git. I am a noob to php and am looking to enhance my coding abilities.

**Comment by mGm-Lizard** [comment]
Why isn't there a Correction_php.php that hasn't been corrected? Without an uncorrected file available, it doesn't seem like it'd be helpful for anyone.

**Comment by P1kachu** [comment]
You must check the exercices sheets, these a not corrected.

**Pull request merged**